### ISSUE TITLE
navigation error by not use the installed plugins

### ISSUE THREAD

**Issue by vallemar** [nativescript-vue/nativescript-vue#1018]
When you navigate to a new page I get this warn because the plugin is not being passed to the newly created app. I have created in my local a small store of the plugins that are used and if it is applied in each createApp the error disappears. Warn:
```
[Vue warn]: injection "Symbol(pinia)" not found.
```

**Comment by rigor789** [comment]
Should be fixed by 3e5c9f168d42959af9d2d67ea2704b8b2fe38ad8

**Comment by vallemar** [comment]
@rigor789 can you open this issue, now it doesn't complain about pinia but i have errors in plugins

```
  [$navigateTo] Failed to navigate:
  Error: Element for MapView already registered.
  If this is intentional set 'overwriteExisting: true' in 'meta' Error: Element for MapView already registered.
  If this is intentional set 'overwriteExisting: true' in 'meta'

```

**Comment by rigor789** [comment]
I'm guessing `MapView` is registered through `app.use()`? Well, I guess we can remove/set overwriteExisting to true by default - can't think of a better way to handle this... Because right now it's working as expected, all the plugins you applied to the root instance, are also applied to all navigation apps, so plugins will registerElement every time 🤔 

Seems redundant in most cases...

**Comment by vallemar** [comment]
Yes, I know, I think the same thing but there are many components that do this use() syntactic sugar just to register the component. In any case, even the components that do more things should also be updated so that it doesn't fail anyway... If this is not a performance problem, I would set the override to true by default so as not to heat our heads over this...

**Comment by rigor789** [comment]
I'm worried about plugins that perhaps do more than just `registerElement` in `.use` - those might break when called multiple times as well 🤔 

Will need to think a bit more about this, perhaps navigate/modals shouldn't create new apps, or at least make them "sub-apps" of the root app or something like that instead...

**Comment by vallemar** [comment]
Could you make a function render (h()) that renders the component you navigate to?

**Comment by camilo02021** [comment]
I'm having the same issue when I use "$navigateTo" when click on button I got " Error: Element for MapView already registered." It is because I'm using the plugging like this: import GoogleMaps from '@nativescript/google-maps/vue' and then  const app = createApp(Home)
app.use(GoogleMaps).start(), 
It is how the documentation says, I think  the "use.()" and the "$navigateTo"  they conflict.
Any solution for this?

**Comment by vallemar** [comment]
@camilo02021 you can register MapView as:
```ts
import {createApp, registerElement} from 'nativescript-vue';

registerElement("MapView", () => require('@nativescript/google-maps').MapView)

...
```

**Comment by camilo02021** [comment]
Excellent, very helpful, it works, it happens only with vue3, In vue2 I can
use it by "Vue.use(GoogleMaps)"
